libraries/botframework-connector/src/auth/userTokenClientImpl.ts

```diff
@@ -44,6 +62,14 @@ export class UserTokenClientImpl extends UserTokenClient {
         return result._response.parsedBody;
     }
 
+    /**
+     * A Task<TResult> representing the result of the asynchronous operation.
```

Comment:
Tasks are a .NET thing. In JS we have Promises. Also, that should be the return information, not the description of the function.